src/main/cli/settings.c

```diff
@@ -1516,7 +1516,7 @@ const clivalue_t valueTable[] = {
 #endif
 
 // PG_VCD_CONFIG
-#if defined(USE_MAX7456) || defined(USE_FRSKYOSD)
+#if defined(USE_MAX7456) || defined(USE_FRSKYOSD) || defined(USE_MSP_DISPLAYPORT)
```

Comment:
I think at this point we should probably split these settings - `vcd_h_offset` and `vcd_v_offset` seem to be only supported by the MAX7456 driver.